src/Simulation/qdk_sim_rs/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// The following two attributes include the README.md for this crate when
+// building docs (requires +nightly).
+// See https://github.com/rust-lang/rust/issues/82768#issuecomment-803935643
+// for discussion.
+#![cfg_attr(doc, feature(extended_key_value_attributes))]
+#![cfg_attr(doc, cfg_attr(doc, doc = include_str!("../README.md")))]
+// Set linting rules for documentation. We will stop the build on missing docs,
+// or docs with known broken links. We only enable this when all relevant
+// features are enabled, otherwise the docs build will fail on links to
+// features that are disabled for the current build.
+#![cfg_attr(all(doc, feature = "python"), deny(rustdoc::broken_intra_doc_links))]
+#![cfg_attr(doc, deny(missing_docs))]
+#![cfg_attr(doc, warn(missing_doc_code_examples))]
```

Comment:
Of note, this causes quite a few warnings to be logged for missing code examples, which is good but distracting for now. I don't think this needs to be resolved for this PR, but worth calling out as folks will see those warnings pretty prominently during local development (you already explained this one to me offline, just noting here for tracking/follow-up).

Reply:
I'll add a comment, as you suggest.